framework/src/st_things/Kconfig

```diff
@@ -27,13 +27,6 @@ config ST_THINGS_ARTIK_HW_CERT_KEY
 	bool "Enable Artik H/W Certificate & Key"
 	depends on ARCH_BOARD_ARTIK05X_FAMILY
 	select TLS_WITH_SSS
-	select SUPPORT_FULL_SECURITY
-	select HW_RNG
-	select HW_DH_PARAM
-	select HW_ECDH_PARAM
-	select HW_RSA_VERIFICATION
-	select HW_ECDSA_VERIFICATION
-	select HW_RSA_ENC
```

Comment:
These configurations are related with hardware acceleration, which need to be arranged soon.  What do you think about keeping them alive, so that we can easily arrange at once in the near future?

Reply:
I'll keeping it for future arrange.